eng/Directory.Build.Data.props

```diff
@@ -7,6 +7,26 @@
       SemVer 2.0 is supported by NuGet since 3.0.0 (July 2015) in some capacity, and fully since 3.5.0 (October 2016).
     -->
     <NoWarn>$(NoWarn);NU5105</NoWarn>
+    <!--
+      Disable some FxCop rules
+     -->
+    <NoWarn>
+      $(NoWarn);
```

Comment:
Should we file an issue to turn these back on someday when we have a chance to clean everything up?

Reply:
Yes, I will do that

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/6763